src/useq/_mda_sequence.py

```diff
@@ -401,3 +479,42 @@ def iter_sequence(sequence: MDASequence) -> Iterator[MDAEvent]:
             sequence=sequence,
             global_index=global_index,
         )
+        global_index += 1
+
+
+def _iter_position_sequence(
```

Comment:
not actually an iterator (just returns a single value) ... so would want a different name, but it will be removed anyway, so no biggie